packages/material-ui/src/Stepper/Stepper.test.js

```diff
@@ -167,4 +167,14 @@ describe('<Stepper />', () => {
       );
     });
   });
+
+  it('renders with a null child', () => {
+    const wrapper = shallow(
+      <Stepper>
+        <Step />
+        {null}
+      </Stepper>,
+    );
+    assert.strictEqual(wrapper.find(Step).length, 1);
```

Comment:
This test passes without the implementation change. What should be asserted is that there is no connector if I understood the original issue.

Reply:
The test is just here to add a missing test case not really related to this pull request. The pull request on it's own is solving an advanced test case.

Reply:
I start to think that we should be labeling this issue "won't fix" but keep the new test case given the simple workaround.

Reply:
Maybe "has workaround" is more constructive in this case since it also provides a solution.

Reply:
Yeah, plus, it's breaking the test case. Let's keep it simple.